tests/benchdnn/dnnl_memory.cpp

```diff
@@ -137,11 +137,28 @@ int execute_reorder(const dnn_mem_t &src, dnn_mem_t &dst,
     }
 #endif
 
+    dnnl_status_t status;
     if (!r_pd_) {
-        DNN_SAFE(dnnl_reorder_primitive_desc_create(&r_pd_, src.md_,
-                         src.engine(), dst.md_, dst.engine(), attr),
-                WARN);
+        status = dnnl_reorder_primitive_desc_create(
+                &r_pd_, src.md_, src.engine(), dst.md_, dst.engine(), attr);
+        if (status != dnnl_success
+                && dnnl_memory_desc_equal(src.md_, dst.md_)) {
+            // If fail to create reorder pd, use plain data copy.
+            const int64_t chunk_size = 64;
+            const int64_t n_chunks = div_up(src.nelems(), chunk_size);
+            benchdnn_parallel_nd(n_chunks, [&](int64_t idx_chunk) {
+                int64_t idx_start = idx_chunk * chunk_size;
+                int64_t idx_end = MIN2(idx_start + chunk_size, src.nelems());
+                for (int64_t idx = idx_start; idx < idx_end; ++idx) {
+                    float e = src.get_elem(idx);
+                    dst.set_elem(idx, e);
+                }
+            });
+            return OK;
+        } else
+            return FAIL;
```

Comment:
Suggest to build the block like that:
```
while (!r_pd_) {
    // Fallback to GPU reorder.
    dnnl_status_t status = dnnl_reorder_primitive_desc_create(
            &r_pd_, src.md_, src.engine(), dst.md_, dst.engine(), attr);
    if (status != dnnl_success) break;

    // code that executes pd (written below)
}

// Last resort: fallback to a plain copy for identical mds.
// new code goes here. If not identical, return FAIL.
```
Please add `BENCHDNN_PRINT(2, "%s\n", "[REORDER] Fallback to plain copy.");` when the fallback succeeded, to see what's going on.